SparkPost/support-docs
# DEN-31 - Add support page for setting up SSO

### What Changed

- Added page for configuring SSO in Analyst web app

### How To Test or Verify

- Verify instructions are clear and images make sense.
- Check spelling and grammar

### PR Checklist

#### All PRs Checklist


#### Content Changes Checklist
